.github/workflows/coverage.yml

```diff
@@ -16,6 +16,20 @@ jobs:
                     MYSQL_ROOT_HOST: "%"
                 ports:
                     - 13306:3306
+            psgql:
```

Comment:
Supposed to be `pgsql` and not `psgql` right?

Reply:
Dammit... copy/pasted but only fixed test... yeah, you're right.  Technically doesn't matter for anything, but I'll fix that